scipy/stats/tests/test_distributions.py

```diff
@@ -371,6 +371,51 @@ def test_pdf_zero(self):
         assert_equal(stats.geninvgauss.pdf(2e6, 50, 2), 0)
 
 
+class TestGenHyperbolic(object):
+    def setup_method(self):
+        np.random.seed(1234)
+
+    def test_pdf_t(self):
+        # Test Against T-Student with 1 - 100 df
+        res = []
+        for df in np.linspace(1,100,20):
```

Comment:
```suggestion
        for df in np.linspace(1, 100, 20):
```
(PEP8, as mentioned above)